index.js

```diff
@@ -1,44 +1,55 @@
 const readline = require('readline');
 const rl = readline.createInterface(process.stdin, process.stdout);
 
-
-
 function ask(questionText) {
   return new Promise((resolve, reject) => {
     rl.question(questionText, resolve);
   });
 }
-
-let range = [...Array(100).keys()].map(i => ++i);
-function getRandomNumber(number) {
-  return range[Math.floor(Math.random() * range.length)];
+function randomInteger(min, max) {
+  let range = max - min + 1;
+  return min + Math.floor(Math.random() * range);
 }
-let filterHigher = range.filter(guessedNumber => guessedNumber > secretNumber);
-console.log(filterHigher)
+//function getMedian(min, max) {
+//  return Math.round((max + min) / 2);
+//}
 
 start();
-
 async function start() {
+  let min = 0;
+  let max = 100;
+  let guess = randomInteger(min, max);
+
 
   console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
   let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
   console.log('You entered: ' + secretNumber);
- 
-  let guessedNumber = await ask(`Is the number ${getRandomNumber()}? `);
-    while (guessedNumber !== 'yes') {
-      let higherOrLower = await ask('Is it higher or lower? ');
-      if (higherOrLower == 'higher') { 
-        await ask(`Is the number ${getRandomNumber()}? `);
-      } else if (higherOrLower == 'lower') {
-        await ask(`Is the number ${getRandomNumber()}? `);
-       } else {
-        console.log('Yes that is the answer!');
-        return;
-       }   
-      }}
-      
-    
-  
-
-    
-  //process.exit();
+
+  let guessedNumber = await ask('Is this your number ' + guess + '? ');
+  let guessCounter = 1;
+
+  while (guessedNumber !== 'yes') {
```

Comment:
consider broadening our allowable inputs. If you convert the `guessedNumber` to a consistent capitalization using `.toLowerCase()` here your input will no longer be case sensitive.